banjo/conf/schema.xml

```diff
@@ -218,6 +218,7 @@
    <field name="work_id" type="long" indexed="true" stored="true"/>
    <field name="work_pi" type="string" indexed="true" stored="true"/>
    <field name="work_alias_pi" type="text_general" indexed="true" stored="true" multiValued="true"/>
+   <field name="work_alias_sort" type="string" indexed="true" stored="true"/>

```

Comment:
@MingtaoSun just realise that work alias would have to be system generated, so the update looks fine.